src/App.tsx

```diff
@@ -1,39 +1,50 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import React from 'react';
 import './App.scss';
+import { Clock } from './Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+export class App extends React.Component {
```

Comment:
```suggestion
export class App extends React.Component<Props, State> {
```
before component create interfaces or types for component props and state